### PR TITLE
Rework OpenTelemetry overview page

### PR DESCRIPTION
**Pull request by jack-berg**
Rework the OpenTelemetry overview page, which was written several years ago and was in need of some spring cleaning. Summary of changes:

- Rework and combine "OpenTelemetry -> Overview -> Intro to monitoring OpenTelemetry", "OpenTelemetry -> Overview -> Reference Architecture" into a single "OpenTelemetry -> Overview" page.
- Remove "OpenTelemetry -> Feature comparison" from the nav, but retain link to it from "OpenTelemetry -> Overview"

### CODE REVIEW

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-introduction.mdx [hunk 1]
```diff
@@ -15,428 +15,108 @@ redirects:
...
-* Exporters that enable you to send data to a backend of your choice
+* A [specification](https://opentelemetry.io/docs/specs/otel/) of language agnostic APIs for instrumenting the core pillars of observability (traces, metrics, logs), and SDKs for configuring how telemetry is exported out of process, with implementations exist in [11+](https://opentelemetry.io/docs/languages/) languages. There is a substantial catalog of instrumentation available built on top of these APIs.
+* [OTLP](https://opentelemetry.io/docs/specs/otlp/), a general-purpose telemetry data delivery protocol.
+* [Semantic conventions](https://opentelemetry.io/docs/specs/semconv/) describing the shape of telemetry data for common domains (i.e. HTTP servers, messaging systems, and many more).
+* The [collector](https://opentelemetry.io/docs/collector/), a highly configurable and extensible data collection and processing pipeline used to monitor infrastructure and enrich, filter, and otherwise transform telemetry.
 
-The components of OpenTelemetry work together to create some distinct advantages for capturing telemetry data:
+These components work together to create distinct advantages:
```
Comment (jack-berg) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-introduction.mdx:37]: I've rewritten the rows in this table with content I think better reflects the state of things today.

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-introduction.mdx [hunk 2]
```diff
@@ -15,428 +15,108 @@ redirects:
...
-As OpenTelemetry matures, we will continue to support new OpenTelemetry data models and provide you curated UI experiences in our platform.
-
-### New Relic agents [#apm-agents]
-
-In general, New Relic's <InlinePopover type="apm" /> and infrastructure agents will collect more telemetry data for your services and hosts. Plus, they offer a wide range of configuration options and an extensive set of auto-instrumentation capabilities.
-
-New Relic's APM agents offer detailed transaction trace visibility for individual services. They also offer predefined sampling to balance the performance impact of your instrumentation against the need to capture enough data to gain helpful insights.
-
-<Callout variant="important">
-  <DoNotTranslate>**Running New Relic agents and OpenTelemetry tooling in the same process**</DoNotTranslate>
```
Comment (jack-berg) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-introduction.mdx:155]: Moved this callout to "OpenTelemetry -> Get started -> OpenTelemetry APM Monitoring -> Overview"

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-introduction.mdx [hunk 3]
```diff
@@ -15,428 +15,108 @@ redirects:
...
-  unbeknownst to the other and independently generate telemetry from your
-  application. At worst, they’ll stomp on each other leading to unpredictable
-  behavior.
-
-  New Relic cannot make any assurances that our APM agents are compatible with
-  a different APM product running in the same process. We recommend you pick the
-  one–and only one–that best fits your needs.
-</Callout>
-
-## How OpenTelemetry works with New Relic [#how-it-works]
```
Comment (jack-berg) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-introduction.mdx:178]: This content is superseded by "OpenTelemetry -> OpenTelemetry data in New Relic"

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-introduction.mdx [hunk 4]
```diff
@@ -15,428 +15,108 @@ redirects:
...
-        Supported types: string, boolean, int, double, bytes
-      </td>
-    </tr>
+New Relic aims to have first class support for OpenTelemetry, including ingesting all OTLP data into our general purpose observability platform and building user experiences on top of OpenTelemetry data to help drive insights from the data out of the box. As components emerge and develop our platform capabilities will evolve alongside. However, please be cognizant of the maturity status of any OpenTelemetry components you are integrating with. While we try to stay on top of changes, it can be challenging to build around breaking changes of experimental components.
 
-    <tr>
-      <td>
-        LogRecord attributes
-      </td>
+## OpenTelemetry reference architecture [#reference-architecture]
```
Comment (jack-berg) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-introduction.mdx:110]: I moved the ref architecture page to a section here. The architecture diagram itself still needs some updating, but even in its current state, I think its useful to put a picture to words.

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-introduction.mdx [hunk 5]
```diff
@@ -15,428 +15,108 @@ redirects:
...
+We recommend that you explore both New Relic and OpenTelemetry options. With New Relic instrumentation, there are inherent advantages to developing instrumentation and platform features which work together, and New Relic integrations tend to work better out of the box. On the other side, OpenTelemetry offers an unparalleled degree of flexibility and control, but this sometimes requires additional research and effort to achieve the desired outcome.
 
-For more details, see the [logs information in our best practices guide](/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts#logs).
+For a closer look at the differences between OpenTelemetry and New Relic instrumentation, see our [detailed comparison](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison).
```
Comment (jack-berg) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-introduction.mdx:102]: Here's the link to only link to the comparison page previously available at "OpenTelemetry -> Overview -> Feature comparison". I considered deleting it right now, but the content on this page needs some needs some attention, but there are occasional valuable bits of insight, so I figured its better to keep it available but less prominent. We need to revisit the content in a followup.

Comment (jack-berg) [reply src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-introduction.mdx]: Discussed in meeting and decided to remove the comparison and this link to it and come back to it later.

**Review by ghost — APPROVED** [review]

**Pull request merged**